packages/ui/src/molecules/PaymentMethods/stories/assets/Icons.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
```

Comment:
What do you think about moving this file to stories/Icons.tsx?

Reply:
Thanks for bringing this up. I realized that there is no standard for this in the components that are being created. I think it's a topic for discussion within the squad.

Reply:
We discussed this in the Self Service Squad daily and it was decided that we will keep this way.